test/test-built/test-built-exports.js

```diff
@@ -41,9 +41,10 @@ describe("built file exports", function () {
       expect(heartbeat[k]).to.be.a(mytype);
     }
   });
+  let numRecipes = 3;
```

Comment:
I like the instinct here.  I am not sure that I like the solution.  

Claim:  adding a recipe should be 'hard' and a bit of a hassle. I have no good reason for rejecting this, other than searching for the error message string will fail here.